include/diy/partners/common.hpp

```diff
@@ -113,8 +113,8 @@ fill(int round, int gid, std::vector<int>& partners) const
   {
     partner += step;
     coords[kv.dim] = partner;
-    int partner_gid = Decomposer::coords_to_gid(coords, divisions_);
-    partners.push_back(partner_gid);
+    int loop_partner_gid = Decomposer::coords_to_gid(coords, divisions_);
```

Comment:
Why rename this variable? I don't necessarily disagree, if there is a good reason.

Reply:
I'll change it to just reuse the existing variable instead.